### ISSUE TITLE
Review token limits & access to discord channels

### ISSUE THREAD

**Issue by tamarandom** [commons-stack/iteration0#319]
## 📋   Anything to add?

(Carry over from #242)
Token limits & access - including CS team members/contributors
-->Acceptance email still a bit unclear; Revise acceptance email - still not quite clear w/ incentives

## 🎉   Subtasks

- [ ] 
- [ ] 
- [ ] 

## 🤼‍♂️   Reviewer

@

**Comment by jessicazartler** [comment]
@knobsdao @GriffGreen @krisjpi  discussing this in a meeting today - please note here when you finish
@Jeff-Emmett @jessicazartler @tamarandom @liviade1 if you have any thoughts on this please comment here

**Comment by knobsdao** [comment]
The process is very organic, with peer-validation among reviewers and a baseline of what could increase or decrease the limit. Could be informed by a framework like this one: https://blog.blablacar.com/trust (pretty sure there must be more current ones being used out there)

**Comment by knobsdao** [comment]
Related to #110. Refine.

**Comment by knobsdao** [comment]
Hi @krisjpi, assigning this one to you now. Here is the information on CSTK holdings I presented today: https://docs.google.com/spreadsheets/d/1oGOeI2-haEHFmJvug1XNTpufiJV94TLiY1mrKWbD-qM/edit?usp=sharing

The folks over 10K can be a category. And we could group the other categories so as not to create a crazy amount of new gated channels. I added the actual numbers (without personal info) to this doc as a reference. Took off the registry max-trust data.
